internal/gcsx/random_reader.go

```diff
@@ -47,15 +54,27 @@ const maxReadSize = 8 * MB
 // Minimum number of seeks before evaluating if the read pattern is random.
 const minSeeksForRandom = 2
 
+// Constants for read types - sequential/random
+const sequential = "Sequential"
+const random = "Random"
+
 // Initialize the metrics.
 func init() {
-	v := &view.View{
-		Name:        "gcsfuse_read_bytes",
-		Measure:     readBytes,
-		Description: "The number of bytes read from GCS",
-		Aggregation: view.Sum(),
-	}
-	if err := view.Register(v); err != nil {
+	if err := view.Register(
```

Comment:
nit: formatting looks off.

Reply:
It looks proper in the intellij: https://screenshot.googleplex.com/FAhYcbCsieQ9t5z.png

Reply:
I see it as the attached image, hence the comment.

![codeformat](https://user-images.githubusercontent.com/1674877/183272677-0e92da07-3c63-4400-844f-71279535729e.png)
t